### PR TITLE
ADD missing scripts to RPMs

### PR DESCRIPTION
**Pull request by fgalan**
It solves #1083. In additon, I have take this opportunity to solve #2.

### CODE REVIEW

File: rpm/SPECS/contextBroker.spec [hunk 1]
```diff
@@ -117,8 +117,10 @@ rm $RPM_BUILD_ROOT/usr/share/contextBroker/tests/cases/CMakeLists.txt
 cp LICENSE $RPM_BUILD_ROOT/usr/share/doc/contextBroker
 cp scripts/testEnv.sh test/functionalTest/testHarness.sh test/functionalTest/testDiff.py test/functionalTest/harnessFunctions.sh $RPM_BUILD_ROOT/usr/share/contextBroker/tests
 cp scripts/accumulator-server.py $RPM_BUILD_ROOT/usr/share/contextBroker/tests 
+cp scripts/httpsPrepare.sh $RPM_BUILD_ROOT/usr/share/contextBroker/tests
```
Comment (LeandroGuillen) [inline rpm/SPECS/contextBroker.spec]: We have the script, but I don't think it's reachable yet.

For example, take this line in `0000_https_support/https.test`:
    
    ${SCRIPT_HOME}/../../scripts/httpsPrepare.sh --keyFileName /tmp/harnessTest.key --certFileName /tmp/harnessTest.pem

If the test is run from `/usr/share/contextBroker/tests` then this points to `/usr/scripts/httpsPrepare.sh`.

Comment (kzangeli) [inline rpm/SPECS/contextBroker.spec]: I guess the easiest way to fix this is to call httpsPrepare.sh using $PATH.
We'll all need to add "~/git/fiware-orion/scripts" (or wherever we have the repo) to our PATH.
The problem arises when we use more than one repo (not that httpsPrepare.sh will change between repos, but still).

Another way would be to create an ENV VAR pointing to the file.

No good way of solving this, really

Comment (LeandroGuillen) [inline rpm/SPECS/contextBroker.spec]: how about putting httpsPrepare.sh in the same directory as harnessFunctions and testHarness? It is only used there.

Comment (kzangeli) [inline rpm/SPECS/contextBroker.spec]: Sure, let's do that

Comment (fgalan) [inline rpm/SPECS/contextBroker.spec]: I'll do that in this PR.

Comment (fgalan) [inline rpm/SPECS/contextBroker.spec]: As in 8eb1e70d1521a9fc3ad9fb122beaa14346d3a8cc this has been fixed.

**Comment by LeandroGuillen** [comment]
LGTM

**Pull request merged**